Library/Dnn.PersonaBar.UI/admin/personaBar/scripts/permissionGrid.js

```diff
@@ -27,7 +38,20 @@ if (typeof dnn.controls === "undefined" || dnn.controls === null) { dnn.controls
                     'css!../../../../../Resources/Shared/Components/Tokeninput/Themes/token-input-facebook.css'
             ]);
         },
-
+        _getPermissionsDefinition: function () {
+            var permissionsDefinitions = this.data.permissionDefinitions;
+            var permissionsMap = {};
+            permissionsDefinitions.forEach(function (curr) {
+
+                var label = curr.permissionName.toUpperCase().replace(' ', '_');

```

Comment:
`data.permissionDefinitions` does not include the string identifier which we have in the database?

Reply:
@mean2me you should rely on PermissionCode, not PermissionName

Reply:
> I feel safer if I normalize those strings

but you are not safer. for all I know, the names can change and the `PermissionKey` does not

Reply:
I agree, guys, it means I need to change the API since permissionKey is not returned by *GET /API/personaBar/assets/GetFolderDetails?folderId=<ID>*